enhancements/sig-policy/72-content-to-ansilbe-job/README.md

```diff
@@ -0,0 +1,174 @@
+# Providing Context to the Ansible Automation for a Policy Violation 
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in
+      [website](https://github.com/open-cluster-management-io/open-cluster-management-io.github.io/)
+
+## Summary
+
+The Open Cluster Management (OCM) Policy add-on Ansible integration should pass the additional information 
+into an Ansible job when triggering the creation. It means that the Ansible user can get more policy violation 
+information and use them in the Ansible playbook.
+
+## Motivation
+
+Policy Automation uses the `policyRef` field to match the target policy and monitor its non-compliance status. 
+A non-compliance policy can trigger its Policy Automation to create an Ansible job on the Ansible Tower, which 
+launches a predefined Ansible playbook against an inventory of hosts. During this creation, we can pass the 
+additional information to the Ansible job as `extra_vars`.
+
+The current limitation is that we only pass the non-compliant cluster as `extra_vars`, which isn't enough to 
+describe the policy violation status. The following fields could be added:
+1. The violated policy name
+2. The policy violation message
+3. The replicated policy status for each non-compliant cluster
+
+
+### Goals
+
+Adding the name of the violated policy, the violation message, and the replicated policy status for each 
+non-compliant cluster into the Ansible Job.
+
+### Non-Goals
+
+- None at this time
+
+## Proposal
+Policy automation runs on the hub cluster, it should not get the propagated policy status from the managed cluster,
+which requires Search or Observability dependency. The same design also applies to the hub of the cluster scenario.
+
+### New Content Structure
+
+Current `extra_vars` has a field `target_clusters` contains non-compliant cluster name list:
+```json
+{
+  "target_clusters": [
+    "cluster1", 
+    "cluster2", 
+    "cluster3",
+  ]
+}
+```
+
+The new content structure will pass more information:
+```json
+{
+  "target_clusters": [
+    "cluster1", 
+    "cluster2", 
+    "cluster3",
+  ],
+  "policy_name": "policy1",
```

Comment:
I'm thinking it might be useful if we also included the policy namespace as an extra var in case the same policy name is used in multiple places.

Reply:
yes, the namespace is needed.  A nice to have would be the acm hub clustername: example `policy-grc-cp-dev-lq5qm.dev08.red-chesterfield.com`.  I use it in my examples but I would guess it may be available some other way.